src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="people" element={<PeoplePage />}>
+          <Route path=":slug" element={<PeoplePage />} />
+        </Route>
+        <Route path="*" element={<NotFoundPage />} />
+        <Route path="home" element={<Navigate to="/" />} />
```

Comment:
should be replaced in history
![image](https://github.com/mate-academy/react_people-table-basics/assets/22824431/4d6763ac-0eea-487a-8a90-56d1317d1d6d)